### PR TITLE
Fix long titles in breadcrumbs

### PR DESCRIPTION
**Pull request by dipikabh**
### Description

The longer title of the Glossary appears in the breadcrumbs. The part after the colon is not particularly necessary or helpful and is explained in the intro para on the [Glossary landing page](https://developer.mozilla.org/en-US/docs/Glossary).

### Motivation

To reduce the glossary title in the breadcrumbs

### Additional details

Before:

<kbd>screenshot with the longer title MDN Web Docs Glossary: Definitions of Web-related terms in the breadcrumbs</kbd>

After:
<kbd>screenshot with the shorter title MDN Web Docs Glossary in the breadcrumbs</kbd>

### CODE REVIEW

**Comment by dipikabh** [comment]
Perhaps there is also a case to further shorten the title to "MDN Glossary"

**Review by wbamberg — COMMENTED** [review]
I agree that this is better but would also vote for just "Glossary" as even better :).

**Comment by hamishwillee** [comment]
FWIW I agree ^^^

**Comment by dipikabh** [comment]
Thanks everyone for your feedback!
I noticed that for [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS), we're using the shortened title in the breadcrumbs while keeping the landing page title unchanged. I believe updating the breadcrumb would be a better approach to get the intended result. The same approach could then also be applied to the [SVG](https://developer.mozilla.org/en-US/docs/Web/svg) breadcrumbs.

**Comment by wbamberg** [comment]
Can we do that in content using `short-title`? It seems like a shame to hack it inside Yari.

File: files/en-us/glossary/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "MDN Web Docs Glossary: Definitions of Web-related terms"
+title: MDN Web Docs Glossary
 short-title: MDN Web Docs Glossary
```
Comment (caugner) [inline files/en-us/glossary/index.md:3]:
```suggestion
title: "MDN Web Docs Glossary: Definitions of Web-related terms"
short-title: Glossary
```

File: files/en-us/web/svg/element/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: SVG element reference
+short-title: Elements
```
Comment (dipikabh) [inline files/en-us/web/svg/element/index.md:3]: The SVG breadcrumbs currently shows `References > SVG: Scalable Vector Graphics > SVG element reference > <animate>`.

The breadcrumbs for HTML elements is already shortened. Following the same style for SVG elements by adding a `short-title`. 

<kbd></kbd>

**Review by estelle — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**